Add tutorial 1-7: Coordinate Systems

Vassalware opened the pull request:
This PR adds the OpenTK port of the LearnOpenGL Coordinate Systems tutorial.

devvoid commented:
Looks fantastic! No complaints from me.

Perksey commented on learn/chapter1/7-coordinate-systems.md:
```diff
@@ -0,0 +1,292 @@
+
+As you can see, due to perspective the lines seem to coincide the farther they're away. This is exactly the effect perspective projection tries to mimic and it does so using a perspective projection matrix. The projection matrix maps a given frustum range to clip space, but also manipulates the w value of each vertex coordinate in such a way that the further away a vertex coordinate is from the v ...
+
+![Perspective Division](img/7-perspective_division.png)
```
I'm getting a 404 on this image.

Perksey commented on learn/chapter1/7-coordinate-systems.md:
```diff
@@ -0,0 +1,292 @@
+> **Right-handed system**
+> By convention, OpenGL is a right-handed system. What this basically says is that the positive x-axis is to your right, the positive y-axis is up and the positive z-axis is backwards. Think of your screen being the center of the 3 axes and the positive z-axis going through your screen towards you. The axes are drawn as follows:
+> ![Right-Handed Coordinate System](img/7-coordinate_systems_right_handed.png)
+>To understand why it's called right-handed do the following:
```
Needs another blank line to ensure that it's not on the same line as the image.
![image](https://user-images.githubusercontent.com/11160611/54943686-b6c30b80-4f29-11e9-8df7-3ac8990930e3.png)

Vassalware commented:
Fixed those issues. Looks like the vscode markdown preview extension I was using isn't as accurate as the stock one; I wasn't getting the text on the same line as the image until I switched back.

And don't misunderstand: I didn't write the article. It's almost identical to the source content at http://learnopengl.com, just edited slightly to use C# and OpenTK.
